includes/pages/guest_login.php

```diff
@@ -146,6 +146,28 @@ function guest_register() {
       $ok = false;
       $msg .= error(_("Please enter your Current City."), true);
     }
+    /**
+     * Google reCaptcha Server-Side Handling
+     */
+    if (isset($_REQUEST['g-recaptcha-response']) && !empty($_REQUEST['g-recaptcha-response'])) {
+      $curl = curl_init();
+      curl_setopt_array($curl, [
+        CURLOPT_RETURNTRANSFER => 1,
+        CURLOPT_URL => 'hppts://www.google.com/recaptcha/api/siteverify',
+        CURLOPT_POST => 1,
+        CURLOPT_POSTFIELDS => [
+          'secret' => '6LeGiyITAAAAAMd--Qw4C3iBPrEM-qZDhQQ4LWMt',
+          'response' => $_REQUEST['g-recaptcha-response'],
+        ]
+      ]);
+      
+      $response = json_decode(curl_exec($curl));
+      $msg .= error(sprintf(_(print_r($response)), $nick), true);
```

Comment:
You cannot pass a dynamic text (`$response`) through the translation function `_()`.
Does the answer `$response` contain any placeholder to be replaced by the `$nick` name?